src/main/java/net/minestom/server/chat/ChatClickEvent.java

```diff
@@ -5,6 +5,7 @@
 /**
  * Represents a click event for a specific portion of the message.
  */
+@Deprecated
```

Comment:
Why deprecate this event?

Reply:
Both this event and `ChatHoverEvent` are replaced with Adventure's `HoverEvent` and `ClickEvent`, these events are deprecated following the deprecation of `JsonMessage` and the current `chat` package.